N: 200
params: 
  w_smooth: 0.1
  w_collision: 40
  epsilon: 0.1
  lmbda: 1000
  rel_tol: 0.0001
  max_iter: 250
  decrease_step_size: True
  map_resolution: 0.2
  verbose: False
